src/main.cpp

```diff
@@ -14,6 +16,16 @@ static void handleSignal(int signum);
 
 int main(int argc, char **argv)
 {
+	int pid = fork();
+	if (pid < 0)
+		return EXIT_FAILURE;
+	if (pid == 0){
+		char *file = strdup("/app/updateHTML.sh");
+		char *args[] = {file, NULL};
+		execve(file, args, NULL);
+		return EXIT_FAILURE;
+	}
+
```

Comment:
It's a good debugging feature, but I'm not sure if it's necessary to hard code tests like this on the main branch. In the future, these kinds of features will be directly supported by the CGI.

Also, be careful, this creates an infinite while loop that consumes resources even when the page is not requested (which won't be the case with the CGI).

Lastly, from a technical standpoint, strdup is not secure; it's preferable to use a C++ string with c_str(). The child process is also never waited for or killed, which leads to process leaks (very dangerous).

That's why I think it's better to keep this feature on a debug branch and not on the main branch.

Reply:
This is not a debugging feature
In the latest version, the script is only call when needed (no loop)
strdup seems to be required for this particular usage, but i'm searching if i can do it in other way.

Reply:
OK, but why "hard code" the launch of a bash script from our C++ code when CGIs can handle all .sh files directly?

Reply:
OK, i understand. Let keep this here, and update when CGIs will be implemented !

Reply:
i'm not finished for tonight, i'll request another review later

Reply:
Okay, no problem !